docs/listfilepages/globalfilestagpages/globalfilesprexxx.md

```diff
@@ -0,0 +1,2236 @@
+
+------------------------------------------------------------------------
+
+**<span id="preskill"></span> Tag Name:** PRESKILL:x,y=z,y=z
+
+**Variables Used (x):** Number (Number of skills)
+
+**Variables Used (y):** Text (Skill name)
+
+**Variables Used (y):** TYPE.Text (Skill type)
+
+**Variables Used (z):** Number (Skill ranks)
+
+**What it does:**
+
+-   Sets skill requirements.
+-   If you require multiples of a particular type of skill, you need to include a `      TYPE.Text     ` sub-tag for each instance in the comma delimited list.
+-   Skills types with multiple versions like "Craft (Pottery)" or "Knowledge (Local)" there must be a space between the name and the parentheses because the text must match the entry exactly.
+
+**Examples:**
+
+`     PRESKILL:1,Spot=10,Listen=10    `
+
+Character must have at least 10 ranks in either "Spot" or "Listen".
+
+`     PRESKILL:2,TYPE.Spy=2,TYPE.Spy=2    `
+
+Character must have two "Spy" skills with at least two ranks in each of them.
+
+`     PRESKILL:3,Appraise=3,Decipher Script=3,Knowledge (TYPE=Other)=3    `
+
+Character must have three skills with at least three ranks in each of them.
+
+------------------------------------------------------------------------
+
+**<span id="preskillmult"></span> Tag Name:** PRESKILLMULT:x,y=z,y=z
+
+**Variables Used (x):** Number (The number of skills that must be equal to or greater than the numbers specified for the check to succeed).
+
+**Variables Used (y):** Text (A defined skill name).
+
+**Variables Used (y):** TYPE=Text (A defined skill type).
+
+**Variables Used (z):** Number (The number of ranks the associated skill must meet or exceed).
+
+**What it does:**
+
+-   Similar to PRESKILL.
+-   This tag will set a flag on the character.
+-   It only works as a prereq if the rank in the skill divided by the rank needed is equal to the flag.
+-   The flag starts at 1, but goes up by one every-time the prereq is met. So the first feat would require 'rank' in the skill, the second feat 'rank\*2', and so on. That would support the way regional feats are described in FR.
+-   If you require multiples of that type, you need to add TYPE.x for each instance in the comma delimited list.
+-   This tag is merely a dummy tag that works like PRESKILL so we don't have to redesign the lst files when this tag gets working.
+
+NOTE: This tag has some problems. It will work in some files but not all.
+
+**Example:**
+
+`     PRESKILLMULT:1,Spot=10,Listen=10    `
+
+Character must have a Spot or Listen at 10 or above.
+
+------------------------------------------------------------------------
+
+<span id="preskillsit"></span> \*\*\* New 6.03.00
+
+**Tag Name:** PRESKILLSIT:w,x,y=z,y=z
+
+**Variables Used (w):** Number (Number of skills)
+
+**Variables Used (x):** SKILL=Text (Skill name)
+
+**Variables Used (y):** Text (Situation)
+
+**Variables Used (z):** Number, Variable, or Formula (Skill ranks)
```

Comment:
Skill ranks cannot be correct. SKILLSIT is a virtual copy, I believe the proper term is skill total. @thpr to verify.